src/components/SignUp.js

```diff
@@ -102,7 +102,7 @@ class SignUp extends Component {
 
   handleRegister = async () => {
     const { email, name } = this.state;
-    const checkingEmail = emailValidator(email);
+    const checkingEmail = emailValidator(email.toLowerCase());
```

Comment:
Perhaps add a new variable `lowercaseEmail = email.toLowercase()` and use it in the next lines.